Description: Move boost libraries to Libs.private in pkg-config
Author: Jose Luis Rivero <jrivero@osrfoundation.org>
Last-Update: 2019-07-12

diff -r e7c99f1634db cmake/pkgconfig/gazebo.in
--- a/cmake/pkgconfig/gazebo.in	Thu Jul 11 21:02:00 2019 +0000
+++ b/cmake/pkgconfig/gazebo.in	Fri Jul 12 10:00:23 2019 +0000
@@ -6,5 +6,6 @@
 Description: Gazebo Libraries
 Version: @GAZEBO_VERSION_FULL@
 Requires: sdformat protobuf @TBB_PKG_CONFIG@ ignition-math4 ignition-msgs1 ignition-transport4 @IGNITION_FUEL_TOOLS_PKGCONFIG@
-Libs: -Wl,-rpath,${prefix}/@CMAKE_INSTALL_LIBDIR@/gazebo-@GAZEBO_MAJOR_VERSION@/plugins -L${libdir} -L${prefix}/@CMAKE_INSTALL_LIBDIR@/gazebo-@GAZEBO_MAJOR_VERSION@/plugins -lgazebo_transport -lgazebo_physics -lgazebo_sensors -lgazebo_rendering -lgazebo_gui -lgazebo_client -lgazebo_msgs -lgazebo_common -lgazebo @Boost_PKGCONFIG_LIBS@ @APPLE_PKGCONFIG_LIBS@
+Libs: -Wl,-rpath,${prefix}/@CMAKE_INSTALL_LIBDIR@/gazebo-@GAZEBO_MAJOR_VERSION@/plugins -L${libdir} -L${prefix}/@CMAKE_INSTALL_LIBDIR@/gazebo-@GAZEBO_MAJOR_VERSION@/plugins -lgazebo_transport -lgazebo_physics -lgazebo_sensors -lgazebo_rendering -lgazebo_gui -lgazebo_client -lgazebo_msgs -lgazebo_common -lgazebo @APPLE_PKGCONFIG_LIBS@
+Libs.private: @Boost_PKGCONFIG_LIBS@
